### ISSUE TITLE
Provide option to serve system clock time

### ISSUE THREAD

**Issue by fakuivan** [cturra/docker-ntp#47]
For systems without internet connectivity, it would be nice to have an option to use the system clock as a time source. I've managed (trough command injection :P) to set chrony to use the system clock with `NTP_SERVERS` set to `--env=NTP_SERVERS="$(printf "%s\n%s\n%s" 127.127.1.0 "local stratum 10" "# ")"` which resutls in the following `/etc/chrony/chrony.conf` file:

```
# https://github.com/cturra/docker-ntp

# chrony.conf file generated by startup script
# located at /opt/startup.sh

# time servers provided by NTP_SERVER environment variables.
server 127.127.1.1
local stratum 10
#  iburst

driftfile /var/lib/chrony/chrony.drift
makestep 0.1 3
rtcsync

allow all
```

from my testing, it looks like `local stratum 10` is required for this to work, no idea why.

**Comment by cturra** [comment]
thanks for this suggestion @fakuivan. it's not a use case i've personally run into, but i can see how this may be useful for people.

i've cut a new branch, which you can find below, that adds local support. what the updated script does is looks through the list of time servers provided and if the 127.0.0.0/8 range is found, configures that entry for local statum 10. 

 => https://github.com/cturra/docker-ntp/tree/local-support

i went ahead and did a little testing myself, but before i merge and push to dockerhub, would you be able to give it a test for me also? to do this, just clone the branch above and run through the normal steps you've use in the past to launch this container... but without any of the clever printf() stuff you're doing (smart btw 😉). let me know how you make out.

**Comment by fakuivan** [comment]
Yeah, that works! As a side note, I was thinking more of a switch like `"$USE_LOCAL_CLOCK"=true`, also I'm not sure `127.0.0.0/8` is precisely the correct subnet, I think it's `127.127.0.0/16` where the last octet is related to the stratum and clock type somehow.

Sources: 
 * https://docs.oracle.com/cd/E19109-01/tsolaris8/805-8125/after-time-1/index.html
 * https://serverfault.com/questions/766701/getting-gpsd-address-for-ntpd/766707#766707
 * http://eecis.udel.edu/~mills/ntp/html/refclock.html
 * https://www.meinbergglobal.com/english/info/ntp.htm

**Comment by cturra** [comment]
all of 127.0.0.0/8 is assigned for use as the host loopback address and is non-routable over the wire. more can be found here:

 => https://datatracker.ietf.org/doc/html/rfc5735#section-3

i'd prefer not to add more flags for users, but instead, use existing arguments and help make the correct decision at startup on how to configure the chronyd service.

**Comment by cturra** [comment]
this is now merged into master (see: [PR#48](https://github.com/cturra/docker-ntp/pull/48)) and a new build pushed to dockerhub.

thanks again for your help on this @fakuivan 🙏🏼
